doctoshotgun.py

```diff
@@ -228,19 +241,26 @@ def do_login(self):
 
         return True
 
-    def find_centers(self, where, motives=None):
+    def find_centers(self, where, motives=None, current_page=1):
         if motives is None:
             motives = self.vaccine_motives.keys()
         for city in where:
             try:
-                self.centers.go(where=city, params={'ref_visit_motive_ids[]': motives})
+                if current_page is None or current_page == 1:
+                    params={'ref_visit_motive_ids[]': motives}
+                else:
+                    params={'ref_visit_motive_ids[]': motives, 'page': current_page}
```

Comment:
We could just always send the page number. It works even with `page=1`
```suggestion
                params={'ref_visit_motive_ids[]': motives, 'page': current_page}
```